tracing/otel/helpers.go

```diff
@@ -21,3 +25,36 @@ func SpanRecordError(span trace.Span, err error, description string, eventOption
 	)
 	span.RecordError(err, eventOptions...)
 }
+
+var defaultGCPTraceLog = gcpTraceLog{
+	spanFieldName:  "logging.googleapis.com/spanId",
+	traceFieldName: "logging.googleapis.com/trace",
+}
+
+type gcpTraceLog struct {
+	traceFieldName string
+	spanFieldName  string
+	gcpProjectID   string
+}
+
+// LogFromCtx returns a log from the provided context. It adds GCP trace and span fields so the log can be associated with cloud tracing
+func (tl *gcpTraceLog) LogFromCtx(ctx context.Context) *zerolog.Logger {
```

Comment:
Could also add a test for this function.

Reply:
Yeh I wanted to but got frustrated trying to mock the span. I'll give it another stab

Reply:
Seems the mock tracer lets you specify some hard coded values for the span: https://github.com/open-telemetry/opentelemetry-go/blob/main/bridge/opentracing/internal/mock.go#L46

Reply:
Just ended up mocking my own span which was simple enough